src/server/JasmineGraphServer.cpp

```diff
@@ -3577,7 +3614,16 @@ bool JasmineGraphServer::initiateAggregator(std::string host, int port, int data
         server_logger.log("Received : " + JasmineGraphInstanceProtocol::HANDSHAKE_OK, "info");
         string server_host = utils.getJasmineGraphProperty("org.jasminegraph.server.host");
         write(sockfd, server_host.c_str(), server_host.size());
-        server_logger.log("Sent : " + server_host, "info");
+        server_logger.log("Sent host : " + server_host, "info");
+
+        bzero(data, 301);
```

Comment:
It's better not to change it to 300. It will cause bugs. We have to change the buffer size to 301. We have to keep 301 (with sonarlint bug) and change the definition of `data`.

Reply:
Then, line 3571 should be changed as 
```
char data[FED_DATA_LENGTH+1]; 
```
and this also need to be changed as 
```suggestion
        bzero(data, FED_DATA_LENGTH+1);
```

Reply:
Fixed this issue